charts/ingress-nginx/values.yaml

```diff
@@ -293,7 +293,7 @@ controller:
   ## Enable mimalloc as a drop-in replacement for malloc.
   ## ref: https://github.com/microsoft/mimalloc
   ##
-  enableMimalloc: false
+  enableMimalloc: true
```

Comment:
@aledbf was there any special reason to enable mimalloc by default as part of raising min requirement to k8s 1.16 APIs?
 
If not, wish enabling mimalloc by default was a separate PR (or part of default config changes PR) and had changelog entry. I guess changelog entry can be added still for previous release.